test/per_file_coverage.sh

```diff
@@ -79,6 +79,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/server/admin:97.4"
 "source/server/admin:profiler-lib:83"
 "source/server/config_validation:74.8"
+"source/extensions/http/early_header_mutation/header_mutation:95.2" # Death tests don't report LCOV
```

Comment:
Why are we needing death tests for this feature?

Reply:
Just for unset enum of `oneof`. It's ok for me to remove it. 🤔

Reply:
Ah OK that's fine. No problem.

Reply:
https://storage.googleapis.com/envoy-pr/9d19ddc/coverage/source/extensions/http/early_header_mutation/header_mutation/index.html

Yep, the only uncovered lines are PANICs.  Bummer that goes below the limit and 100% fine to adjust for that :-)